### PR TITLE
NEXUS-9019 Adding the invalidate negative cache button

### PR DESCRIPTION
**Pull request by joedragons**
Includes:
* Section within proxies for buttons that can be used for future buttons as well as this one
* Presence of the invalidate negative cache button and description

https://issues.sonatype.org/browse/NEXUS-9019

### CODE REVIEW

File: chapter-configuration.asciidoc [hunk 1]
```diff
@@ -497,7 +497,13 @@ component in subsequent deployments.
 
 ===== Proxy
 
-The configuration for proxy repositories in the 'Proxy' section contains the following parameters:
+In addition to the 'Delete repository' button, a proxy repository has several buttons which can be used to perform actions
```
Comment (mosabua) [inline chapter-configuration.asciidoc]: General prefer to use 'that' over 'which'.

**Comment by mosabua** [comment]
+1 .. good to go. Merge it ;-)

**Comment by mosabua** [comment]
Can you do the other buttons as well? That would maintain the same writing style in this section.

**Comment by joedragons** [comment]
I intended to look at the others if nothing came up.
I didn't want to do more than 1 ticket in a PR.
Thanks, will merge.

**Pull request merged**